.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,7 +3,7 @@
 The following has been addressed in the PR:
 
 * [ ] There is a related issue
-* [ ] All code matches the [style guide](https://github.com/dojo/meta/blob/master/STYLE.md)
+* [ ] All code has been formatted with [`prettier`](https://prettier.io/) as per the [readme code style guidelines](./../#code-style)
```

Comment:
I guess this would be automatic though?  So maybe we can drop this?

Reply:
It should be automatic (commit hook), but figured no harm in leaving it?

Reply:
Something that can be easily revisited in the future if we come up with a better template.